src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx

```diff
@@ -131,6 +136,18 @@ function AppointmentsPage({
           }}
         />
       )}
+
+      {showCheetahScheduleButton && (
+        <ScheduleNewProjectCheetah
+          startNewAppointmentFlow={() => {
+            recordEvent({
+              event: `${GA_PREFIX}-schedule-project-cheetah-button-clicked`,
+            });
+            startNewAppointmentFlow();
```

Comment:
I think we made need to make an project cheetah specific version of `startNewAppointmentFlow`

Reply:
I do this in a ticket I am working on now for the calendar POC